lib/plugins/wallet/cardano/api.js

```diff
@@ -0,0 +1,223 @@
+const BN = require('../../../bn')
+
+const unitScale = 6
+
+class CardanoAPI {
+  static get INCLUDE_INCOME_TRANSACTIONS() {
+    return 1 >> 1
```

Comment:
These are certainly wrong. `1>>1` and `1>>2` is 0. Also, why is there need to have this as a function? a constant would be enough?

Reply:
Fixed to `1 << 0` & `1 << 1`. JS classes don't have constants - we can define consts outside this classes, but I want a connection between them. I can do this by `CardanoAPI.INCLUDE_INCOME_TRANSACTIONS = 1 << 0`, but this isn't constant at all.

Reply:
ou, static properties works in node :+1: sorry, i was confused, becase in browser this now only works with transpilers